proto/regen/ecocredit/v1/state.proto

```diff
@@ -30,8 +30,8 @@ message CreditType {
   uint32 precision = 4;
 }
 
-// ClassInfo represents the high-level on-chain information for a credit class.
-message ClassInfo {
+// Class represents the high-level on-chain information for a credit class.
+message Class {
```

Comment:
have we evaluated whether or not this will collide with clients in other languages? i tested it locally with python and typescript, so seems like its clear there, but not sure if other langs might have an issue as `Class` is usually a keyword in other OO langs.

Reply:
cc: @aaronc @ryanchristo